sass/_vars.scss

```diff
@@ -1,113 +1,310 @@
+                start: (
+                    light: white,
+                    dark: palette(darkgray, 3),
+                ),
+                end: (
+                    light: palette(lightgray, 1),
+                    dark: palette(darkgray, 4),
+                ),
+            ),
+            hover: (
+                gradient: (
+                    start: (
+                        light: palette(lightgray, 1),
+                        dark: palette(darkgray, 2),
+                    ),
+                    end: (
+                        light: palette(lightgray, 3),
+                        dark: palette(darkgray, 3),
+                    ),
+                ),
+                border: (
+                    light: palette(lightgray, 8),
+                    dark: palette(darkgray, 8),
+                ),
+            ),
+            primary: (
+                gradient: (
+                    start: (
+                        light: palette(blue, 4),
+                        dark: color.scale(palette(blue, 6), $saturation: -10%),
+                    ),
+                    end: (
+                        light: palette(blue, 6),
+                        dark: color.scale(palette(blue, 8), $saturation: -10%),
+                    ),
+                ),
+                hover: (
+                    gradient: (
+                        start: (
+                            light: palette(blue, 3),
+                            dark: color.scale(palette(blue, 5), $saturation: -10%),
+                        ),
+                        end: (
+                            light: palette(blue, 5),
+                            dark: color.scale(palette(blue, 7), $saturation: -10%),
+                        ),
+                    ),
+                ),
+                disabled: (
+                    light: palette(blue, 3),
+                    dark: color.scale(palette(blue, 5), $saturation: -10%),
+                ),
+            ),
+        ),
+        scrollbar: (
+            bg: (
+                default: (
+                    light: palette(lightgray, 6),
+                    dark: palette(darkgray, 4),
+                ),
+                hover: (
+                    light: palette(lightgray, 7),
+                    dark: palette(darkgray, 3),
+                ),
+                active: (
+                    light: palette(lightgray, 8),
+                    dark: palette(darkgray, 1),
+                ),
+            ),
+        ),
+        shadow: (
+            default: (
+                light: palette(lightgray, 8),
+                dark: palette(darkgray, 7),
+            ),
+            inset: (
+                light: palette(lightgray, 9),
+                dark: palette(darkgray, 7),
+            ),
+            subtle: (
+                light: palette(lightgray, 5),
+                dark: palette(darkgray, 6),
+            ),
+            focus: (
+                default: palette(indigo, 5),
```

Comment:
Does this also need updating?

Reply:
No, the single definition will be used for both themes and the variable will be "shadow-focus", but I see how it's confusing. I adjusted the map functions to strip the default key from the variable name.

The default could also be left out, but then the indentation would look a bit messy.